include/genn/genn/code_generator/backendBase.h

```diff
@@ -326,6 +326,12 @@ class GENN_EXPORT BackendBase
     //! Get hash digest of this backends identification and the preferences it has been configured with
     virtual boost::uuids::detail::sha1::digest_type getHashDigest() const = 0;
 
+    //! Generate code to push a profiler range marker
+    virtual void genPushProfilerRange(CodeStream &os, const std::string &name) const {}
```

Comment:
```suggestion
    virtual void genPushProfilerRange(CodeStream&, const std::string&) const {}
```
to fix warnings in GCC